examples/src/main/scala/sttp/tapir/examples/security/csrfTokens.scala

```diff
@@ -0,0 +1,186 @@
+// {cat=Security; effects=Future; server=Pekko HTTP}: Securing endpoint with CSRF tokens example
+
+//> using dep com.softwaremill.sttp.tapir::tapir-core:1.11.12
+//> using dep com.softwaremill.sttp.tapir::tapir-pekko-http-server:1.11.12
+//> using dep com.softwaremill.sttp.client3::core:3.10.2
+
+package sttp.tapir.examples.security
+
+import org.apache.pekko.actor.ActorSystem
+import org.apache.pekko.http.scaladsl.Http
+import org.apache.pekko.http.scaladsl.server.Route
+import sttp.client3.*
+import sttp.model.{HeaderNames, StatusCode}
+import sttp.model.headers.{Cookie, CookieValueWithMeta, WWWAuthenticateChallenge}
+import sttp.shared.Identity
+import sttp.tapir.*
+import sttp.tapir.model.*
+import sttp.tapir.generic.auto.*
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.ServerEndpoint.Full
+import sttp.tapir.server.pekkohttp.*
+
+import java.util.UUID
+import scala.concurrent.duration.*
+import scala.concurrent.{Await, Future}
+import scala.util.matching.Regex
+
+case class User(name: String)
+
+case class ProtectedUser(user: User, csrfToken: UUID)
+
+// Simple in-memory users database. Do not keep passwords in clear text in real projects!
+object Users {
```

Comment:
since we're in scala3, lets go braceless

this should be reformatted using scalafmt, do you have this enabled?

Reply:
Formatting done but `object` with methods cannot be braceless - does not compile

Reply:
They can, use : or google ;)